backend/pom.xml

```diff
@@ -69,6 +69,12 @@
             <artifactId>java-jwt</artifactId>
             <version>4.3.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>RELEASE</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Po co dodałeś TestNG? Przecież i tak testów nie piszecie. Jak już coś to pisz testy po prostu w junit